orm/join_test.go

```diff
@@ -23,6 +26,8 @@ type HasOne struct {
 
 	HasOne   *HasOne
 	HasOneId int
+
+	CustomKey string `pg:"custom_key"`
```

Comment:
Please don't change existing tests - otherwise it is not clear whether the old code is working the same or not. This should be a new separate test or an example.

Reply:
@vmihailenco is it ok to just duplicate old test and modify it?

Reply:
Yes, that is perfectly fine :+1: